src/status_im/contexts/wallet/events.cljs

```diff
@@ -502,3 +502,34 @@
          activities           (cske/transform-keys transforms/->kebab-case-keyword activities)
          sorted-activities    (sort :timestamp activities)]
      {:db (assoc-in db [:wallet :activities] sorted-activities)})))
+
```

Comment:
there should be somewhere more specific to add these events. 
This file is already very large and these events are specific to adding an account. 

Perhaps status-im.contexts.wallet.add-account.events is a better location for these 👍